test/extensions/filters/http/ext_authz/ext_authz_integration_test.cc

```diff
@@ -320,6 +328,14 @@ TEST_P(ExtAuthzGrpcIntegrationTest, HTTP2DownstreamRequestWithLargeBody) {
   expectCheckRequestWithBody(Http::CodecClient::Type::HTTP2, 2048);
 }
 
+// Verifies that the request body is included in the CheckRequest when the downstream protocol is
+// HTTP/2 and the size of the request body is larger than max_request_bytes.
+/*
+TEST_P(ExtAuthzGrpcIntegrationTest, HTTP2DownstreamRequestWithVeryLargeBody) {
```

Comment:
?

Reply:
This is a test that I added and failed to work as I've expected it to. This is one of the tests I'm still working on.